UnityProject/Assets/Scripts/Systems/FilthGenerator/FilthGenerator.cs

```diff
@@ -36,13 +36,13 @@ public class FilthGenerator : ItemMatrixSystemInit
 		private int filthGenerated = 0;
 		public int FilthCleanGoal { get; private set; } = 0;
 
-		public override void Initialize()
+		public override async UniTask Initialize()
```

Comment:
all this stuff basically needs to be tested in build

Reply:
that's why we have staging.

Reply:
well, some testing saves having to do a quick  revert  and having to redo code